flow-report/tsconfig.json

```diff
@@ -1,14 +1,37 @@
 {
-  "extends": "../tsconfig",
```

Comment:
I'll switch this back to extending and move most of this back to a `tsconfig-base.json` or whatever in a follow up